apps/server/src/shared/repo/mongo.patterns.ts

```diff
@@ -0,0 +1,8 @@
+export class MongoPatterns {
+	/**
+	 * Regex to escape strings before use as regex against database.
+	 * Used to remove all non-language characters except numbers, whitespace or minus.
+	 */
+	static REGEX_MONGO_LANGUAGE_PATTERN_WHITELIST =
```

Comment:
alphanumeric instead of language?

Reply:
language means also these alpha with special chars added